e2e/motoko/Main.mo

```diff
@@ -147,7 +147,7 @@ shared ({ caller = installer }) actor class Main() {
                 await canister.eth_getTransactionCount(
                     ethMainnetSource,
                     {
-                        address = "0xdAC17F958D2ee523a2206206994597C13D831ec7";
+                        address = "0x1789F79e95324A47c5Fd6693071188e82E9a3558";
```

Comment:
CI was occasionally running into request size limits for BlockPI, so this adjusts the test to use a less active smart contract address (ckStagingETH instead of ckETH).